packages/go/bomenc/normalize_test.go

```diff
@@ -288,59 +198,52 @@ func TestNormalizeToUTF8_LargeInput(t *testing.T) {
 		encFrom  Encoding
 	}
 
-	// Generate a large input with 1000 Unicode code points
-	var utf16LE, expected []byte
-
-	for i := 0; i < 1000; i++ {
-		r := rune(i % 0x10FFFF) // Use all possible Unicode code points
-
-		// UTF-16
-		u16 := utf16.Encode([]rune{r})
-		for _, c := range u16 {
-			utf16LE = append(utf16LE, byte(c), byte(c>>8))
-		}
-
-		// Expected UTF-8
-		buf := make([]byte, 4)
-		n := utf8.EncodeRune(buf, r)
-		expected = append(expected, buf[:n]...)
+	// Generate a large data set with 1000 Unicode code points
+	// NOTE: We don't want to begin the payload with a NULL character because it would then be impossible to discern between UTF16LE and
```

Comment:
```suggestion
	// NOTE: We don't want to begin the payload with a NULL character because it would then be impossible to discern between UTF16LE and UTF32LE
```
nit: seems the comment got cut off early